### PR TITLE
[Merged by Bors] - feat(Data/Opposite): Show that op, unop are surjective

### PR DESCRIPTION
**Pull request by uniwuni**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Opposite.lean [hunk 1]
```diff
@@ -59,6 +59,10 @@ theorem op_injective : Function.Injective (op : α → αᵒᵖ) := fun _ _ => c
 theorem unop_injective : Function.Injective (unop : αᵒᵖ → α) := fun ⟨_⟩⟨_⟩ => by simp
 #align opposite.unop_injective Opposite.unop_injective
 
+theorem op_surjective : Function.Surjective (op : α → αᵒᵖ) := fun x => ⟨unop x, rfl⟩
```
Comment (eric-wieser) [inline Mathlib/Data/Opposite.lean:62]: If you put this on line 95, the proof is `equivToOpposite.surjective`

Comment (eric-wieser) [reply Mathlib/Data/Opposite.lean:62]: I think there's still a weak reason relating to defeq that justifies it existing.

Comment (uniwuni) [reply Mathlib/Data/Opposite.lean]: Rewrote both, the other one via `symm`, hope that's fine

**Comment by eric-wieser** [comment]
bors merge

Thanks! Though I'm kind of surprised that you actually need this lemma

**Comment by uniwuni** [comment]
> Thanks! Though I'm kind of surprised that you actually need this lemma

honestly I don't anymore after a refactor, but it seems plausible that I'll need it soon so it doesn't hurt for sure, thanks for the review

**Pull request closed without merging**